ml-agents/mlagents/trainers/torch/action_log_probs.py

```diff
@@ -5,6 +5,38 @@
 from mlagents.trainers.torch.utils import ModelUtils
 
 
+class LogProbsTuple:
```

Comment:
If this is identical to ActionTuple in functionality, but you want to make sure that they aren't accidentally used interchangeably, you could  make them both derive from  e.g. `_ActionTupleBase`